DOCS/man/vo.rst

```diff
@@ -488,10 +488,15 @@ Available video output drivers are:
         Mode ID to use (resolution and frame rate).
         (default: 0)
 
-    ``--drm-overlay=<number>``
-        Select the DRM overlay index to use.
-        Overlay index is zero based, and related to crtc.
-        (default: 0)
+    ``--drm-osd-plane-id=<number>``
+        Select the DRM planed index to use for OSD.
+        Index is zero based, and related to crtc.
+        (default: first overlay plane)
+
+    ``--drm-video-plane-id=<number>``
```

Comment:
Sorry about the nitpicking, but we should probably mention that this is only really the case when we use drmprime hwdec and --vo=gpu --gpu-context=drm, and that otherwise (this includes --vo=drm) the video plane is unused with the osd plane instead being the "video" plane in this case.

I know it's a bit late, but I've been thinking that maybe different naming for the planes could be used here to make the matter less confusing. Maybe something like "drm-render-plane-id" (for what is now "drm-osd-plane-id") and "drm-hwdec-video-plane-id" (for what is now "drm-video-plane-id"), for example. (Although this particular naming would of course not be forward-compatible if we decide to use separate planes for video/OSD in the software decoded case in the future.) 

What do you think (and if anyone has better naming ideas please say)?

Reply:
I see what you're saying. Although we have to consider that options should have a not too much technical name. hwdec probably don't mean anything for most users typically :)
But as you say, in some cases both video & osd are on the same plane (in which case btw you probably don't want any of those options, primary plane will be used then).
Maybe mentioning that those options should be used in the case you mentioned would be enough to clarify that case ?

Reply:
Mentioning that the options should primarily be used together with drmprime hwdec, and that in other cases (--vo=drm, --gpu-context=drm swdec) everything will be rendered to the "osd plane" should be sufficient, yes.

Re: the names: yeah, my names were just examples. I think maybe better ones than that could be thought of.